test/__snapshots__/index.spec.js.snap

```diff
@@ -0,0 +1,42 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The bar library should render correctly the cozy-bar 1`] = `
+<Provider
+  store={
+    BarStore {
+      "appsList": Array [],
+      "claudyActions": null,
+      "helpLink": "",
+      "installedApps": Array [],
+      "settingsAppURL": "",
+      "settingsData": null,
+    }
+  }
+>
+  <Provider
+    store={
+      Object {
+        "dispatch": [Function],
+        "getState": [Function],
+        "replaceReducer": [Function],
+        "subscribe": [Function],
+        Symbol(Symbol.observable): [Function],
+      }
+    }
+  >
+    <Connect(I18n)
+      dictRequire={[Function]}
+    >
+      <Unknown
```

Comment:
Unknown ?

Reply:
I know it's not great :/, but that tests is just to check the bar first root rendering (it was broken by some previous PRs). When I will have more time I will try to improve them, but I am opened for some improvement ideas :)

Reply:
I think the time is now, you will never have more time.